docs/AA--Javascript-in-EE/components/form/select/event-select.md

```diff
@@ -0,0 +1,136 @@
+# `<EventSelect />`
```

Comment:
same points I made above for the DatetimeSelect component docs apply to this one